examples/login_node_example/implement_login_node.yml

```diff
@@ -0,0 +1,20 @@
+# Copyright 2021 Dell Inc. or its subsidiaries. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+---
+
+- name: Create user and assign slurm permission
+  hosts: manager
+  gather_facts: false
+  roles:
+    - login_user
```

Comment:
Why exactly are the examples setup as a playbook? If all the example does is show admins how to create a user using FreeIPA, then let's just include the example in markdown in `/docs`